include/dlaf/eigensolver/tridiag_solver/rot.h

```diff
@@ -211,8 +254,9 @@ void applyGivensRotationsToMatrixColumns(comm::Communicator comm_row, comm::Inde
       // would require a tag ad hoc ensuring that communication between same ranks do not get mixed
       // (in addition to having a tag ensuring that other calls to this algorithm do not get mixed too).
       tt::sync_wait(
-          di::whenAllLift(ex::when_all_vector(std::move(comm_checkpoints))) |
-          di::transform(di::Policy<DefaultBackend_v<D>>(), [rot, m, col_x, col_y](auto&&... ts) {
+          di::whenAllLift(ex::just(1),  // workaround for problem with unwrapping
```

Comment:
Conversely, I think all arguments are senders here:
```suggestion
          ex::when_all(ex::just(1),  // workaround for problem with unwrapping
```
Alternatively:
```suggestion
          di::whenAllLift(1,  // workaround for problem with unwrapping
```